packages/viewport/src/index.js

```diff
@@ -187,3 +187,40 @@ export function subscribeIsDesktop( listener ) {
 export function getWindowInnerWidth() {
 	return isServer ? SERVER_WIDTH : window.innerWidth;
 }
+
+/******************************************/
+/*       Vertical Scroll Experiment       */
+/******************************************/
```

Comment:
Should we annotate this here? An A/B test is an application-side logic, but `@automattic/viewport` is a general package that only . Putting it another way: would these three newly-added functions become useless outside of the vertical scroll experiment?

Reply:
So this is something that I would attribute to a cleanup first strategy, So in the event of a cleanup this would be easier and it serves as a boundary in where nobody should touch this code because it most probably will be refactored out.